nixos/modules/services/misc/gitlab.nix

```
@@ -279,6 +279,18 @@ in {
 
       packages.pages = mkPackageOption pkgs "gitlab-pages" { };
 
+      packages.gitlab-rails = mkOption {
+        type = types.package;
+        default = gitlab-rails;
+        description = "The gitlab-rails package to use.";
```

solution suggested by waffle8946 (tested and working):

```suggestion
        default = gitlab-rails;
        defaultText = "";
        description = "The gitlab-rails package to use.";
```